aws-ssm-parameter/aws-ssm-parameter.json

```diff
@@ -78,15 +78,18 @@
     "primaryIdentifier": [
         "/properties/Name"
     ],
-    "readOnlyProperties": [
-        "/properties/Name",
-        "/properties/Type",
-        "/properties/Value"
+    "writeOnlyProperties": [
+        "/properties/Tags",
+        "/properties/Description",
+        "/properties/Tier",
+        "/properties/AllowedPattern",
+        "/properties/Policies"
     ],
     "handlers": {
         "create": {
             "permissions": [
                 "ssm:PutParameter",
+                "ssm:AddTagsToResource",
```

Comment:
Yes looks like this AddTagsToResource is inexplicitly called. If remove this create handler will fail with `Failed to execute remote function: {User: arn:aws:sts::526204198394:assumed-role/CT-aws-ssm-parameter-execution-rol-CtExecutionRole-T4KF5IOQM2Z0/081ad21d-15e9-402c-9303-0ce12369ce94 is not authorized to perform: ssm:AddTagsToResource on resource: arn:aws:ssm:us-west-2:526204198394:parameter/ContractTestParameter because no session policy allows the ssm:AddTagsToResource action (Service: Ssm, Status Code: 400, Request ID: 07cd97d4-5963-45c0-931a-eaab0499714d)}`